docs/content/integrations/embedded-elt.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: "Dagster Embedded ELT"
+description: Lightweight ELT with Dagster
```

Comment:
Can you add some more info here? Maybe even just use the first sentence in the intro for this - it's a good, short and sweet description of what this is.